src/main/snapshot.ts

```diff
@@ -191,6 +191,7 @@ export async function downloadSnapshot(
     if (token.reason === "clear-cache") {
       throw new ClearCacheException();
     } else {
+      console.log(JSON.stringify(error));
```

Comment:
Is `error`  encoded as JSON?

Reply:
Yes. [(ref)](https://stackoverflow.com/a/18391400)

Reply:
fixed: e353296